src/ui/settings/index.js

```diff
@@ -24,20 +24,26 @@ export default function setupSettings(kbnServer, server, config) {
   }
 
   function userSettingsNotFound(kibanaVersion) {
-    if (server.plugins.elasticsearch.status.state === 'green') {
-      server.plugins.kibana.status.red(`Could not find user-provided settings for this version of Kibana (${kibanaVersion})`);
-    } else {
-      server.log(['warning', 'settings'], 'User-provided settings were requested before the Kibana index was ready');
-    }
+    server.plugins.kibana.status.yellow(`Could not find user-provided settings for this version of Kibana (${kibanaVersion})`);
     return {};
   }
 
+  function resetKibanaPluginStatusIfNecessary(user) {
+    const isElasticsearchPluginGreen = server.plugins.elasticsearch.status.state === 'green';
+    const isKibanaPluginCurrentlyYellow = server.plugins.kibana.status.state === 'yellow';
+    if (isElasticsearchPluginGreen && isKibanaPluginCurrentlyYellow) {
```

Comment:
Yeah, good point Lee. I really think all we ought to do when user settings are not found is set the Elastiscearch plugin status to yellow, because ultimately that's the plugin responsible for (re)creating the `.kibana` index and going green again when the index is ready. I think we shouldn't muck with the Kibana plugin status at all. Thoughts?